content/tutorials/react-admin-panel-with-ably-and-refine.textile

```diff
@@ -0,0 +1,289 @@
+---
+authors:
+- author_bio: ''
+  author_image: https://avatars.githubusercontent.com/u/104967037?s=460&v=4
+  author_name: refine
+  author_profile_url: https://github.com/refinedev/refine
+category:
+- channels
+date_published: '2017-02-02T21:32:25+00:00'
+excerpt: Learn how to use refine with Ably to make a realtime admin panel in React
+group: sdk
+index: 102
+languages:
+- javascript
+last_updated: '2023-04-28T11:32:41+00:00'
+level: medium
+platform: browser
+reading_time: 25
+section: tutorials
+tags:
+- refine
+title: Building a realtime React admin panel with Ably and refine
+meta_description: "How to build an admin panel in React using Ably and refine"
+meta_keywords: "Ably realtime, refine, admin panel"
+---
+
+Building complex, data-intensive realtime full-stack applications from the ground up can be daunting and time-consuming. Fortunately, there are several front-end frameworks in the react ecosystem that can improve your development speed and reduce time to production. One such framework is "refine":https://github.com/refinedev/refine.
+
+refine is a react-based framework for building data-intensive applications such as admin panels and dashboards. It comes with built-in features for routing, authentication, and i18n. It supports most popular UI frameworks, design systems, and cloud-based platforms like Firebase, Supabase, and "Ably":https://ably.com.
+
+"Ably":https://ably.com/ is a cloud-based platform that can help you provide reliable, low-latency realtime experience. It has realtime data streaming capabilities like chat, data broadcast, notifications, and more.
+
+When looking to build a realtime data-intensive application, refine and Ably can get you up and running quickly. This tutorial will introduce you to refine and Ably by building a simple realtime React admin panel.
+
+h2. What is Ably
```

Comment:
Thoughts on whether we need this page given this is on our site?

Reply:
I'd say probably we can remove it, good point!